scripts/variantstore/wdl/GvsTieoutPgenToVcf.wdl

```diff
@@ -0,0 +1,163 @@
+version 1.0
+
+
+workflow GvsTieoutPgenToVcf {
+    input {
+        String vcf_extract_extract_task_call_root
+        String pgen_extract_merge_pgen_workflow_root
+    }
+
+    call Tieout {
+        input:
+            vcf_extract_extract_task_call_root = vcf_extract_extract_task_call_root,
+            pgen_extract_merge_pgen_workflow_root = pgen_extract_merge_pgen_workflow_root,
+    }
+}
+
+task Tieout {
+    input {
+        String vcf_extract_extract_task_call_root
+        String pgen_extract_merge_pgen_workflow_root
+        String plink_download_url = "https://s3.amazonaws.com/plink2-assets/plink2_linux_x86_64_20240318.zip"
```

Comment:
@mcovarr One drive-by comment/FYI: I have had similar links to these assets embedded in my pgen scripts for a while, and they break regularly because they seem to reorganize the structure and move things around every few weeks. They broke my scripts twice in the last month because these links don't appear to be stable. I finally wound up [checking them in](https://github.com/broadinstitute/pgen-jni/commit/de500ad710ce70a8171fd3d0806857a1f36d8a2e) to the PGEN repo so I could rely on them.

Reply:
Thank you Chris, I will try this. 👍